<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="LTSG000726" name="cucumberOptions.TestNGTestRunner" tests="35" failures="4" timestamp="2022-02-25T13:30:42 SGT" time="249.801" errors="0">
  <testcase name="runScenario" time="10.632" classname="cucumberOptions.TestNGTestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="6.182" classname="cucumberOptions.TestNGTestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="4.788" classname="cucumberOptions.TestNGTestRunner">
    <failure type="java.lang.AssertionError" message="expected [200] but found [202]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [202]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at stepDefinitions.APIStepDefinition.validate_the_response(APIStepDefinition.java:65)
at ✽.Validate the Response(file:///C:/Elsa/TechChallenge/TheOPPENHEIMER/src/test/java/features/MultipleInsertionAPI.feature:8)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="7.938" classname="cucumberOptions.TestNGTestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="6.614" classname="cucumberOptions.TestNGTestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="15.696" classname="cucumberOptions.TestNGTestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="5.635" classname="cucumberOptions.TestNGTestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="6.078" classname="cucumberOptions.TestNGTestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="17.581" classname="cucumberOptions.TestNGTestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="4.585" classname="cucumberOptions.TestNGTestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="6.753" classname="cucumberOptions.TestNGTestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="4.101" classname="cucumberOptions.TestNGTestRunner">
    <failure type="java.lang.AssertionError" message="expected [200] but found [202]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [202]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at stepDefinitions.APIStepDefinition.validate_the_response(APIStepDefinition.java:65)
at ✽.Validate the Response(file:///C:/Elsa/TechChallenge/TheOPPENHEIMER/src/test/java/features/SingleUserInsertAPI.feature:8)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="9.269" classname="cucumberOptions.TestNGTestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="4.729" classname="cucumberOptions.TestNGTestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="6.268" classname="cucumberOptions.TestNGTestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="6.643" classname="cucumberOptions.TestNGTestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="6.468" classname="cucumberOptions.TestNGTestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="3.792" classname="cucumberOptions.TestNGTestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="7.259" classname="cucumberOptions.TestNGTestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="6.522" classname="cucumberOptions.TestNGTestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="6.809" classname="cucumberOptions.TestNGTestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="8.707" classname="cucumberOptions.TestNGTestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="7.821" classname="cucumberOptions.TestNGTestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="6.600" classname="cucumberOptions.TestNGTestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="4.765" classname="cucumberOptions.TestNGTestRunner">
    <failure type="java.lang.AssertionError" message="expected [200] but found [202]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [202]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at stepDefinitions.APIStepDefinition.validate_the_response(APIStepDefinition.java:65)
at ✽.Validate the Response(file:///C:/Elsa/TechChallenge/TheOPPENHEIMER/src/test/java/features/SingleUserInsertAPI.feature:8)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="6.430" classname="cucumberOptions.TestNGTestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="6.201" classname="cucumberOptions.TestNGTestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="7.062" classname="cucumberOptions.TestNGTestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="8.531" classname="cucumberOptions.TestNGTestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="6.305" classname="cucumberOptions.TestNGTestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="8.341" classname="cucumberOptions.TestNGTestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="7.073" classname="cucumberOptions.TestNGTestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="7.125" classname="cucumberOptions.TestNGTestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="6.253" classname="cucumberOptions.TestNGTestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="4.245" classname="cucumberOptions.TestNGTestRunner">
    <failure type="java.lang.AssertionError" message="expected [200] but found [202]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [202]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at stepDefinitions.APIStepDefinition.validate_the_response(APIStepDefinition.java:65)
at ✽.Validate the Response(file:///C:/Elsa/TechChallenge/TheOPPENHEIMER/src/test/java/features/MultipleInsertionAPI.feature:8)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- cucumberOptions.TestNGTestRunner -->
